tox.ini

```
@@ -7,6 +7,8 @@ envlist =
     py{35,36}-dj111
     py{35,36,37}-dj21
     py{35,36,37,38,39}-dj22
+    py{36,37,38,39}-dj30
+    py{36,37,38,39}-dj31
```

```suggestion
    py{36,37,38,39}-dj{30,31}
```

That said, we should drop support for Python < 3.6 and Django < 2.2, but we can do this in a separate PR.